client/reader/search-stream/index.jsx

```diff
@@ -135,9 +187,22 @@ const FeedStream = React.createClass( {
 	},
 
 	render() {
+		var suggestions = null;
+		var lang = i18nUtils.getLocaleSlug();
+		if ( this.searchSuggestions[lang] ) {
```

Comment:
you probably want to do this work in `getInitialState` and stash it in component state. doing this work every render isn't really necessary.